sphere/run-frontier.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -l
+
+#SBATCH -o %x.o%j
+#SBATCH -A CFD162
+#SBATCH -t 00:30:00
+#SBATCH -q debug
+#SBATCH -S 0
+#SBATCH -N 4
+
+set -e
+
+cmd() {
+  echo "+ $@"
+  eval "$@"
+}
+
+
+SPACK_ENV_NAME=exawind_latest
+MY_EXAWIND_MANAGER=${PROJWORK}/cfd162/${USER}/exawind-manager
+EXAWIND_EXE=exawind
+cmd "module load PrgEnv-amd/8.5.0"
+cmd "module load amd/5.7.1"
```

Comment:
This is inconsistent with https://github.com/Exawind/exawind-manager/blob/b86dc503ba61266197546d78e4a89174c0d3550d/configs/frontier/compilers.yaml#L14

Reply:
Fixed it.